integreat_cms/cms/models/pages/page_translation.py

```diff
@@ -275,7 +275,10 @@ def tags(self) -> list[Promise]:
         if self.page.mirrored_page:
             tags.append(_("Live content"))
 
-        if self.is_empty and self.page.is_leaf():
+        if self.is_empty and (
+            self.page.is_leaf()
+            or not any(self.page.get_non_archived_children(include_parent=False))
```

Comment:
```suggestion
            or not self.page.children.filter(explicitly_archived=False).exists()
```
I think it's unnecessary to query the entire subtree of a page to determine whether it has archived descendants. Shouldn't it be sufficient to look at the direct children, and if no un-archived direct children exist, then we know that all its grand-children are either explicitly or implicitly archived?